airflow/models.py

```diff
@@ -1952,10 +1952,22 @@ class derived from this one results in the creation of a task object,
 
     # For derived classes to define which fields will get jinjaified
     template_fields = []
-    # Defines wich files extensions to look for in the templated fields
+    # Defines which files extensions to look for in the templated fields
     template_ext = []
-    # Defines the extra buttons to display in the task instance model view
-    extra_links = []
+    # Defines which function to call per button
+    # key is the name of the button, which is a string
+    # value is the function to call to get the URL corresponding to the button
+    # The function should follow this format:
+    # type: (BaseOperator, datetime) -> str
```

Comment:
I think you mean TaskInstance? Or do you mean the Operator?

Reply:
The operator